pkg/builder/webhook_test.go

```diff
@@ -570,7 +649,8 @@ func (*TestDefaultValidatorList) DeepCopyObject() runtime.Object   { return nil
 type TestCustomDefaulter struct{}
 
 func (*TestCustomDefaulter) Default(ctx context.Context, obj runtime.Object) error {
-	logf.FromContext(ctx).Info("Defaulting object")
+	log := logf.FromContext(ctx)
```

Comment:
non-blocking nit: I would suggest to revert the formatting

Reply:
Oops, that was me trying to debug some logging issues. Reverted.